app/javascript/article-form/components/ImageUploader.jsx

```diff
@@ -316,16 +318,31 @@ export const ImageUploader = ({
     }
   }
 
+  function formatUploadedImages(links) {
+    let articleImageLinks = "";
+
+        links.forEach(link => {
+          const formattedLink = `![Image description](${link}) \n`;
+
+          articleImageLinks = articleImageLinks + formattedLink;
+        });
+    
+        onImageUploadSuccess?.(articleImageLinks);
```

Comment:
An option here might be to conditionally add a newline only when there are multiple images being uploaded — this will make the tests happier, the test scenario is uploading a single image, and this will keep the output in line with the tests' expectations. (It _might_ also be a good idea to add a test for a multiple image scenario, as well.)

```suggestion
    let articleImageLinks = [];
    links.forEach(link => {
      const formattedLink = `![Image description](${link})`;
      articleImageLinks.push(formattedLink);
    });

    onImageUploadSuccess?.(articleImageLinks.join("\n"));
```